### PR TITLE
Changing execute-test progress bar

### PR DESCRIPTION
**Pull request by AkshathRaghav**
### Description
In #395, it was said that the progress bar present currently is restrictive. #403 revamps that to have a easily adjustable implementation using tqdm. In line with that, this PR changes the execute-test progress bar as well. Only making this PR because #403 is about create-workloads. 

### Issues Resolved
A part of #395 

Here's a sample run: 

```
╰─$ opensearch-benchmark execute-test --pipeline="benchmark-only" \
--workload-path="/home/aksha/Workbench/Workloads/flights" \
--target-host="https://127.0.0.1:9200" \
--client-options="timeout:300,use_ssl:true,verify_certs:false,basic_auth_user:'admin',basic_auth_password:'admin'"

   ____                  _____                      __       ____                  __                         __
  / __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
 / / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
/ /_/ / /_/ /  __/ / / /__/ /  __/ /_/ / /  / /__/ / / /  / /_/ /  __/ / / / /__/ / / / / / / / / /_/ / /  / ,<
\____/ .___/\___/_/ /_/____/\___/\__,_/_/   \___/_/ /_/  /_____/\___/_/ /_/\___/_/ /_/_/ /_/ /_/\__,_/_/  /_/|_|
    /_/

[INFO] Executing test with workload [flights] and provision_config_instance ['external'] with version [2.11.0].

[WARNING] merges_total_time is 496 ms indicating that the cluster is not in a defined clean state. Recorded index time metrics may be misleading.
[WARNING] indexing_total_time is 1231 ms indicating that the cluster is not in a defined clean state. Recorded index time metrics may be misleading.
[WARNING] refresh_total_time is 2687 ms indicating that the cluster is not in a defined clean state. Recorded index time metrics may be misleading.
[WARNING] flush_total_time is 881 ms indicating that the cluster is not in a defined clean state. Recorded index time metrics may be misleading.

Running search: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 100.0/100 [00:06<00:00, 16.58%/s]
Running create-index: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 100.0/100 [00:05<00:00, 19.89%/s]
Running cluster-health: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 100.0/100 [00:05<00:00, 19.90%/s]
Running bulk: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 100.0/100 [00:05<00:00, 19.90%/s]
Running search: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 100.0/100 [00:05<00:00, 19.90%/s]

------------------------------------------------------
    _______             __   _____
   / ____(_)___  ____ _/ /  / ___/_________  ________
  / /_  / / __ \/ __ `/ /   \__ \/ ___/ __ \/ ___/ _ \
 / __/ / / / / / /_/ / /   ___/ / /__/ /_/ / /  /  __/
/_/   /_/_/ /_/\__,_/_/   /____/\___/\____/_/   \___/
------------------------------------------------------
            
|                                                         Metric |   Task |      Value |   Unit |
|---------------------------------------------------------------:|-------:|-----------:|-------:|
|                     Cumulative indexing time of primary shards |        |  0.0213833 |    min |
|             Min cumulative indexing time across primary shards |        |          0 |    min |
|          Median cumulative indexing time across primary shards |        |          0 |    min |
|             Max cumulative indexing time across primary shards |        | 0.00811667 |    min |
|            Cumulative indexing throttle time of primary shards |        |          0 |    min |
|    Min cumulative indexing throttle time across primary shards |        |          0 |    min |
| Median cumulative indexing throttle time across primary shards |        |          0 |    min |
|    Max cumulative indexing throttle time across primary shards |        |          0 |    min |
|                        Cumulative merge time of primary shards |        | 0.00848333 |    min |
|                       Cumulative merge count of primary shards |        |         13 |        |
|                Min cumulative merge time across primary shards |        |          0 |    min |
|             Median cumulative merge time across primary shards |        |          0 |    min |
|                Max cumulative merge time across primary shards |        | 0.00711667 |    min |
|               Cumulative merge throttle time of primary shards |        |          0 |    min |
|       Min cumulative merge throttle time across primary shards |        |          0 |    min |
|    Median cumulative merge throttle time across primary shards |        |          0 |    min |
|       Max cumulative merge throttle time across primary shards |        |          0 |    min |
|                      Cumulative refresh time of primary shards |        |     0.0459 |    min |
|                     Cumulative refresh count of primary shards |        |        283 |        |
|              Min cumulative refresh time across primary shards |        |          0 |    min |
|           Median cumulative refresh time across primary shards |        |          0 |    min |
|              Max cumulative refresh time across primary shards |        |  0.0341667 |    min |
|                        Cumulative flush time of primary shards |        |  0.0146833 |    min |
|                       Cumulative flush count of primary shards |        |         18 |        |
|                Min cumulative flush time across primary shards |        |          0 |    min |
|             Median cumulative flush time across primary shards |        |          0 |    min |
|                Max cumulative flush time across primary shards |        | 0.00993333 |    min |
|                                        Total Young Gen GC time |        |      0.024 |      s |
|                                       Total Young Gen GC count |        |          4 |        |
|                                          Total Old Gen GC time |        |          0 |      s |
|                                         Total Old Gen GC count |        |          0 |        |
|                                                     Store size |        |  0.0108699 |     GB |
|                                                  Translog size |        | 0.00910067 |     GB |
|                                         Heap used for segments |        |          0 |     MB |
|                                       Heap used for doc values |        |          0 |     MB |
|                                            Heap used for terms |        |          0 |     MB |
|                                            Heap used for norms |        |          0 |     MB |
|                                           Heap used for points |        |          0 |     MB |
|                                    Heap used for stored fields |        |          0 |     MB |
|                                                  Segment count |        |         79 |        |
|                                                 Min Throughput |   bulk |    16227.3 | docs/s |
|                                                Mean Throughput |   bulk |    16227.3 | docs/s |
|                                              Median Throughput |   bulk |    16227.3 | docs/s |
|                                                 Max Throughput |   bulk |    16227.3 | docs/s |
|                                        50th percentile latency |   bulk |    112.968 |     ms |
|                                        90th percentile latency |   bulk |    121.081 |     ms |
|                                       100th percentile latency |   bulk |    121.806 |     ms |
|                                   50th percentile service time |   bulk |    112.968 |     ms |
|                                   90th percentile service time |   bulk |    121.081 |     ms |
|                                  100th percentile service time |   bulk |    121.806 |     ms |
|                                                     error rate |   bulk |          0 |      % |
|                                                 Min Throughput | search |     203.22 |  ops/s |
|                                                Mean Throughput | search |     203.22 |  ops/s |
|                                              Median Throughput | search |     203.22 |  ops/s |
|                                                 Max Throughput | search |     203.22 |  ops/s |
|                                        50th percentile latency | search |    16.4081 |     ms |
|                                       100th percentile latency | search |    18.1487 |     ms |
|                                   50th percentile service time | search |    16.4081 |     ms |
|                                  100th percentile service time | search |    18.1487 |     ms |
|                                                     error rate | search |          0 |      % |


--------------------------------
[INFO] SUCCESS (took 39 seconds)
--------------------------------
```
---
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: osbenchmark/utils/io.py [hunk 1]
```diff
@@ -296,38 +303,66 @@ def decompress(zip_name, target_directory):
     elif extension == ".bz2":
         decompressor_args = ["pbzip2", "-d", "-k", "-m10000", "-c"]
         decompressor_lib = bz2.open
-        _do_decompress_manually(target_directory, zip_name, decompressor_args, decompressor_lib)
+        _do_decompress_manually(
+            target_directory, zip_name, decompressor_args, decompressor_lib
+        )
```
Comment (IanHoang) [inline osbenchmark/utils/io.py:308]: This reformatting here and a number of other places doesn't seem necessary. Are you using a plugin?

**Comment by IanHoang** [comment]
@AkshathRaghav A lot of the changes in the diff seem to be reformatting changes that look like they're coming from a plugin. Could you revert the reformatting changes and just include the changes you made?

**Review by IanHoang — CHANGES_REQUESTED** [review]
@AkshathRaghav A lot of the changes in the diff seem to be reformatting changes that look like they're coming from a plugin. To align with best practices in keeping PRs small and to make it easier to review, could you revert the reformatting changes and just include the changes you made for the progress bar?

**Comment by rishabh6788** [comment]
As @IanHoang already mentioned, a lot of formatting changes are making it difficult to spot actual code changes. 
Please disable formatting and update the PR.

**Pull request closed without merging**